website/docs/r/compute_forwarding_rule.html.markdown

```diff
@@ -53,6 +53,12 @@ The following arguments are supported:
 
 * `port_range` - (Optional) A range e.g. "1024-2048" or a single port "1024"
     (defaults to all ports!). Only used for external load balancing.
+  Some types of forwarding target have constraints on the acceptable ports:
```

Comment:
s/target/targets

Also would it make more sense to give a link to where this is documented instead of the full list so we don't have to worry about keeping it up-to-date?

Reply:
Added the missing `s`.

The problem is that the official documentation is currently out of date with the current implementation:
https://cloud.google.com/compute/docs/reference/latest/globalForwardingRules

It is missing ports 1883 and 5222 for TCP and SSL proxy... I already filed a bug with them about it.

Reply:
Oh that's annoying. Ok then.